.github/workflows/pull_request.yml

```
@@ -0,0 +1,28 @@
+on: pull_request
+name: Run Goss tests
+
+jobs:
+  goss_test:
+    name: Goss test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php:
+          # Strings, so they wont be parsed as floats which would
+          # transform "7.0" into "7".
+          - "5.6"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+    steps:
+    - uses: actions/checkout@master
+    - uses: e1himself/goss-installation-action@v1.0.3
```

```suggestion
    - uses: e1himself/goss-installation-action@v1
```

Perhaps just use the latest major version?

---

if we extend the [dependabot](https://github.com/reload/docker-drupal-php7-fpm/blob/master/.github/dependabot.yml) with checking workflow files we should leave it to the patch level and then @dependabot will create pull requests

---

See #46

---

Besides, v1 seems not to be a branch tag: https://github.com/e1himself/goss-installation-action/tree/v1